session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java

```diff
@@ -280,4 +280,59 @@ private void insert_via_sql() throws TException, IoTDBRPCException {
     session.executeNonQueryStatement(
         "insert into root.sg1.d1(timestamp,s1, s2, s3) values(100, 1,2,3)");
   }
+
+  @Test
+  public void checkPathTest()
+      throws ClassNotFoundException, SQLException, IoTDBSessionException, TException, IoTDBRPCException {
+    session = new Session("127.0.0.1", 6667, "root", "root");
+    session.open();
+
+    //test set sg
+    checkSetSG(session, "root.vehicle", true);
+    checkSetSG(session, "root.123456", true);
+    checkSetSG(session, "root._1234", true);
+    checkSetSG(session, "root._vehicle", true);
+    checkSetSG(session, "root.\tvehicle", false);
```

Comment:
how about root.\\tvehicle

Reply:
Don't support it.  The check rules here are the same as those of `TSParser.g`.